tap_github/streams.py

```diff
@@ -32,9 +32,16 @@ def path(self) -> str:  # type: ignore
         """Return the API endpoint path."""
         if "searches" in self.config:
             return "/search/repositories"
-        else:
+        elif "repositories" in self.config:
             # the `repo` and `org` args will be parsed from the partition's `context`
             return "/repos/{org}/{repo}"
+        elif "organizations" in self.config:
+            return "/orgs/{org}/repos"
+        else:
+            raise ValueError(
```

Comment:
Maybe we could raise a ValueError at the beginning already on this condition:
```
if len(set(['repositories', 'organizations', 'searches']).intersection(self.config.keys())) != 1:
  raise ValueError(
      "This tap requires one and only one of the following path options: "
       "search, repositories or organizations"
)
```

Reply:
That makes sense